lib/IRGen/IRGenSIL.cpp

```diff
@@ -1993,54 +1994,131 @@ void IRGenSILFunction::visitGraphOperationInst(GraphOperationInst *i) {
   auto eagerContext = Builder.CreateCall(getContextFn, {});
 
   // Create a TFE op as in:
-  //   auto* op = TFE_NewOp(ctx, "Const", status);
+  //   auto* op = TFE_NewOp(ctx, "OpName", status);
```

Comment:
Doesn't the name have to be unique?

Reply:
In the eager API, the op name is an op type name like "Const", not the op node name (a concept that only exists in the graph mode).